vaadin-item.html

```diff
@@ -12,10 +12,25 @@
   <template>
     <style>
       :host {
-        padding: 4px;
-        white-space: nowrap;
-        text-overflow: ellipsis;
+        display: flex;
+        align-items: center;
+        outline: none;
+        opacity: 0.7;
+        padding: 0.3em 1em;
         box-sizing: border-box;
+        flex-shrink: 0;
```

Comment:
- Remove display and align-items, let’s set those in the core/default styles (not the theme).
- Remove outline and opacity, which are only necessary when used inside a list element.
- Remove padding and box-sizing, let’s keep the default styles to bare minimum for the independent item.
- Remove flex-shrink, it’s only needed for `vaadin-tab`.